apps/bandwagon/forms.py

```diff
@@ -136,6 +135,9 @@ class CollectionForm(ModelForm):
     icon = forms.FileField(label=_lazy(u'Icon'),
                            required=False)
 
+    # This is just a honeypot field for bots to get caught
+    your_name = forms.CharField(widget=forms.HiddenInput(), required=False)
```

Comment:
I think we'd still want a label just to allow for the absolute worst case that CSS wasn't applied case.